docs_versioned_docs/version-ros2humble/ros/config/yaml/overview.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: Robot YAML Overview
+sidebar_label: Overview
+sidebar_position: 1
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+Our robots provide users with a wide range of customization options: sensors, sensor mounting structures, and custom-made parts. Matching the flexibility of our hardware, Clearpath's ROS 2 system is designed to keep all these customization decisions in a single configuration file.
+
+The **Clearpath Robot Configuration YAML**, or `robot.yaml` for short, contains all information pertinent to the entire robot system, allowing robot builders and users to quickly and easily modify any ROS 2 component.
```

Comment:
Change _information pertinent_ to _pertinent information_.

Reply:
Done